processing/src/main/java/io/druid/query/select/SelectQueryQueryToolChest.java

```diff
@@ -182,7 +169,7 @@ public SelectQueryQueryToolChest(ObjectMapper jsonMapper,
 
         final ByteBuffer queryCacheKey = ByteBuffer
             .allocate(
-                1
+                2
```

Comment:
it should be noted in the release notes that this will blow people's cache during rolling upgrades, whether this feature is used or not.